docs/core/testing/unit-testing-mstest-runner-extensions.md

```diff
@@ -40,7 +42,7 @@ When you run your `Microsoft.Testing.Platform` test app with the `--info` switch
 
 This feature is used to understand extensions that would be registering the same command line option or the changes in available options between multiple versions of an extension (or the platform).
 
-## Diagnostic logs
+### Diagnostic logs
 
 The platform produces diagnostic logs that are helpful to understand what is happening during the execution of your test application. The following options are available to configure the produced diagnostic logs:
```

Comment:
This is built-in in platform directly right? Point that out please. I was looking for a nuget to install it through.

Reply:
Maybe we could have built-in section before the packages ones

Reply:
Or maybe instead of listing here I should move to the platfrom option on the intro page and in here only link the other section. WDYT?

Reply:
what you mean by "intro page"? The Overview?

Reply:
I think this is the correct page. Splitting it by how things are shipped does not seem like the right call. When I go here I can learn about a lot of the options that are there, and if they are already built in, then even better. Imho we just need to call out that there is no package to install.